bot/src/config.ts

```diff
@@ -2,4 +2,5 @@ require("dotenv").config();
 
 export let token : string = process.env.TOKEN;
 export let prefix : string = process.env.PREFIX;
-export let owners : string[] = [];
\ No newline at end of file
+export let owners : string[] = ["307478723563749376"];
```

Comment:
What is this hardcoded string?

Reply:
cuma owner dari bot, hanya sementara. nanti kalau sudah ganti kepemilikan, bisa dihapus.